### PR TITLE
[python/it] Translated python to Italian

### PR DESCRIPTION
**Pull request by Ale46**

### CODE REVIEW

File: it-it/python-it.html.markdown [hunk 1]
```diff
@@ -0,0 +1,647 @@
...
+1 + 1  # => 2
+8 - 1  # => 7
+10 * 2  # => 20
+35 / 5  # => 7
+
+# La divisione è un po' complicata. E' una divisione fra interi in cui viene 
+# restituito in automatico il risultato intero.
+5 / 2  # => 2
+
+# Per le divisioni con la virgbola abbiamo bisogno di parlare delle variabili floats.
```
Comment (geoffliu) [inline it-it/python-it.html.markdown]: "virgola"

File: it-it/python-it.html.markdown [hunk 2]
```diff
@@ -0,0 +1,647 @@
...
+# Una stringa può essere considerata come una lista di caratteri
+"Questa è una stringa"[0]  # => 'Q'
+
+# % può essere usato per formattare le stringhe, in questo modo:
+"%s possono essere %s" % ("le stringhe", "interpolate")
+
+# Un nuovo modo per fomattare le stringhe è il metodo format.
+# Questo metodo è quello consigliato
+"{0} possono essere {1}".format("le stringhe", "formattate")
+# Puoi usare della parole chiave se non vuoi contare
```
Comment (geoffliu) [inline it-it/python-it.html.markdown]: "delle"?

File: it-it/python-it.html.markdown [hunk 3]
```diff
@@ -0,0 +1,647 @@
...
+# Copia al contrario della lista
+li[::-1]   # => [3, 4, 2, 1]
+# Usa combinazioni per fare slices avanzate
+# li[inizio:fine:passo]
+
+# Rimuovi arbitrariamente elementi da una lista con "del"
+del li[2]   # li è ora [1, 2, 3]
+# Puoi sommare le liste
+li + altra_li   # => [1, 2, 3, 4, 5, 6]
+# Nota: i valori per li ed _altri_li non sono modificati.
```
Comment (geoffliu) [inline it-it/python-it.html.markdown]: `altri_li`

File: it-it/python-it.html.markdown [hunk 4]
```diff
@@ -0,0 +1,647 @@
...
+# l'ordine non è garantito, anche se a volta può sembrare ordinato
+another_set = set([4, 3, 2, 2, 1])  # another_set è ora set([1, 2, 3, 4])
+
+# Da Python 2.7, {} può essere usato per dichiarare un set
+filled_set = {1, 2, 2, 3, 4}   # => {1, 2, 3, 4}
+
+# Aggiungere elementi ad un set
+filled_set.add(5)   # filled_set è ora {1, 2, 3, 4, 5}
+
+# Fai interazioni su un set con &
```
Comment (geoffliu) [inline it-it/python-it.html.markdown]: "intersezione"

File: it-it/python-it.html.markdown [hunk 5]
```diff
@@ -0,0 +1,647 @@
...
+
+# Funziona da Python 2.6 in su:
+try:
+    # Usa "raise" per generare un errore
+    raise IndexError("Questo è un errore di indice")
+except IndexError as e:
+    pass    # Pass è solo una non-operazione. Solitamente vorrai fare un recupero.
+except (TypeError, NameError):
+    pass    # Eccezioni multiple possono essere gestite tutte insieme, se necessario.
+else:   # Optional clause to the try/except block. Must follow all except blocks
```
Comment (geoffliu) [inline it-it/python-it.html.markdown]: Missed a line here. :)

File: it-it/python-it.html.markdown [hunk 6]
```diff
@@ -0,0 +1,647 @@
...
+
+# Chiamare funzioni con parametri
+aggiungi(5, 6)   # => stampa "x è 5 e y è 6" e restituisce 11
+
+# Un altro modo per chiamare funzioni  è con parole chiave come argomenti
+aggiungi(y=6, x=5)   # Le parole chiave come argomenti possono arrivare in ogni ordine.
+
+
+# Puoi definire funzioni che accettano un numero variabile di argomenti posizionali
+# che verranno interpretati come come tuple se non usi il *
```
Comment (geoffliu) [inline it-it/python-it.html.markdown]: "come come"?

File: it-it/python-it.html.markdown [hunk 7]
```diff
@@ -0,0 +1,647 @@
...
+
+
+####################################################
+## 5. Classi
+####################################################
+
+# Usiamo una sottoclasse da un oggetto per avere una classe.
+class Human(object):
+
+    # Un attributo della classe. E' condiviso da tutte le istanze delle classe
```
Comment (geoffliu) [inline it-it/python-it.html.markdown]: della classe

File: it-it/python-it.html.markdown [hunk 8]
```diff
@@ -0,0 +1,647 @@
...
+    species = "H. sapiens"
+
+    # Costruttore base, richiamato quando la classe viene inizializzata.
+    # Si noti che il doppio leading e l'underscore finali denotano oggetti
```
Comment (geoffliu) [inline it-it/python-it.html.markdown]: Eh... mi sembra strana questa frase, il doppio leading e l'underscore finali... My Italian is rusty, but "il doppio underscore precedenti e finali" sounds better?

File: it-it/python-it.html.markdown [hunk 9]
```diff
@@ -0,0 +1,647 @@
...
+# definiscono un modulo
+import math
+dir(math)
+
+
+####################################################
+## 7. Avanzate
+####################################################
+
+# Generators help you make lazy code
```
Comment (geoffliu) [inline it-it/python-it.html.markdown]: Missed a line.

File: it-it/python-it.html.markdown [hunk 10]
```diff
@@ -0,0 +1,647 @@
...
+def say(say_please=False):
+    msg = "Can you buy me a beer?"
+    return msg, say_please
+
+
+print say()  # Can you buy me a beer?
+print say(say_please=True)  # Can you buy me a beer? Please! I am poor :(
+```
+
+## Ready For More?
```
Comment (geoffliu) [inline it-it/python-it.html.markdown]: Heading here

File: it-it/python-it.html.markdown [hunk 11]
```diff
@@ -0,0 +1,647 @@
...
+
+### Free Online
```
Comment (geoffliu) [inline it-it/python-it.html.markdown]: And here

File: it-it/python-it.html.markdown [hunk 12]
```diff
@@ -0,0 +1,647 @@
...
+* [Automate the Boring Stuff with Python](https://automatetheboringstuff.com)
+* [Learn Python The Hard Way](http://learnpythonthehardway.org/book/)
+* [Dive Into Python](http://www.diveintopython.net/)
+* [The Official Docs](http://docs.python.org/2.6/)
+* [Hitchhiker's Guide to Python](http://docs.python-guide.org/en/latest/)
+* [Python Module of the Week](http://pymotw.com/2/)
+* [A Crash Course in Python for Scientists](http://nbviewer.ipython.org/5920182)
+* [First Steps With Python](https://realpython.com/learn/python-first-steps/)
+
+### Dead Tree
```
Comment (geoffliu) [inline it-it/python-it.html.markdown]: and here

**Comment by geoffliu** [comment]
Thanks for the translation! Just a few minor mistakes, and then we can merge this file.

File: it-it/python-it.html.markdown (hunk 12 above)
Comment (Ale46) [inline it-it/python-it.html.markdown]: With "here" you means only the headings or also the link texts?

Comment (geoffliu) [inline it-it/python-it.html.markdown]: Just heading

**Pull request merged**